src/backend/vulkan/src/physical_device.rs

```diff
@@ -750,6 +752,20 @@ impl PhysicalDevice {
             None
         };
 
+        let display_control = if enabled_extensions.contains(&vk::ExtDisplayControlFn::name()) {
```

Comment:
we need to file an issue to Ash to expose this better, similarly to how `DrawIndirectCount` is exposed